trivy-scanner.py

```diff
@@ -51,19 +53,22 @@
             "MEDIUM": 0,"HIGH": 0,
             "CRITICAL": 0
         }
+        ns = { "namespace": ns_name }
         for item in item_list:
             vuls[item["Severity"]] += 1
-        vul_list[image_name] = vuls
+        vul_list[image_name] = [vuls, ns_name]
```

Comment:
I don't think we should have a namespace label, image name is enough.

Reply:
I need the value of the namespace in the metrics. I will use prometheus to send alert about the vulnerabilities. In my environment multiple teams working in the the same cluster but in different namespaces. I need this to find who has to fix the image.